packages/jest/package.json

```diff
@@ -34,7 +34,7 @@
     "lodash.omit": "^4.5.0"
   },
   "peerDependencies": {
-    "jest": "^24.0.0 || ^25.0.0 || ^26.0.0",
+    "jest": "^24.4.0 || ^25.0.0 || ^26.0.0",
```

Comment:
Bumping the value for a peer dep is a breaking change, so can't be done in Neutrino 9

Reply:
So if we live it as is there is high chance that Jest will work as expected. But it is 99% only. 1% for those users that are on Jest <=24.4.0. May be we can live with this

Reply:
the change is reverted